index.bs

```diff
@@ -1432,8 +1429,131 @@ following additional fields:
 Sessions {#streaming-sessions}
 ------------------------------------
 
-TODO
+To start a streaming session, a sender may send a
+streaming-session-start-request message with the following properties:
 
+: streaming-session-id
+:: Identifies the streaming session.  Must be unique for the (sender,
+    receiver) pair.  Can be used later to modify or terminate a
+    streaming session.
+
+: stats-interval
+:: Indicates the frequency the receiver should send stats messages to
+    the sender.
+
+: stream-offers
+:: Indicates the streams that the receiver can request from the sender.
+
+Each stream offer contains the following properties:
+
+: media-stream-id
+:: Identifies the media stream b	eing offered.  Must be unique within
+    the streaming session.  Can be used by the receiver to request the
+    media session.
+
+: friendly-name
```

Comment:
Nit: display-name, for consistency with agent-info.display-name

Reply:
done